--- node_modules/@ethersproject/experimental/lib/eip1193-bridge.js.orig	2021-10-13 23:59:52.000000000 +0900
+++ node_modules/@ethersproject/experimental/lib/eip1193-bridge.js	2021-10-14 00:00:39.000000000 +0900
@@ -171,7 +171,7 @@
                     case 19: return [4 /*yield*/, this.provider.sendTransaction(params[0])];
                     case 20: return [2 /*return*/, _b.sent()];
                     case 21:
-                        req = ethers_1.ethers.providers.JsonRpcProvider.hexlifyTransaction(params[0]);
+                        req = ethers_1.ethers.providers.JsonRpcProvider.hexlifyTransaction(params[0], { from: true });
                         return [4 /*yield*/, this.provider.call(req, params[1])];
                     case 22: return [2 /*return*/, _b.sent()];
                     case 23:
@@ -208,7 +208,7 @@
                         if (!this.signer) {
                             return [2 /*return*/, throwUnsupported("eth_sendTransaction requires an account")];
                         }
-                        req = ethers_1.ethers.providers.JsonRpcProvider.hexlifyTransaction(params[0]);
+                        req = ethers_1.ethers.providers.JsonRpcProvider.hexlifyTransaction(params[0], { from: true, gas: true });
                         return [4 /*yield*/, this.signer.sendTransaction(req)];
                     case 36:
                         tx = _b.sent();
